tests/test-core.js

```diff
@@ -277,11 +277,11 @@ I have a question. Can you help answer it? –[[User:Novem Linguae|<span style="
 		const revId = 592507;
 		const isBiography = true;
 		const newWikiProjects = [ 'WikiProject Romania' ];
-		const lifeStatus = 'living';
+		const lifeStatus = 'blp';
```

Comment:
Internally in AFCH (but not in the wikicode), can we keep using "living" as the value? I find living to be more readable.